### PR TITLE
Minor tweak to packaging script

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: scripts/appleseed.package.py [hunk 1]
```diff
@@ -446,8 +446,11 @@ def alter_stage(self):
         return
 
     def deploy_stage_to_package_directory(self):
+    	staging_directory = os.path.join(self.settings.package_output_path, "appleseed")
```
Comment (dictoon) [inline scripts/appleseed.package.py]: That should be called `package_directory`.

Comment (dictoon) [reply scripts/appleseed.package.py]: Indentation needs fixing. I'm surprised that Python didn't complain.

**Comment by jdent02** [comment]
Updates in

File: scripts/appleseed.package.py [hunk 2]
```diff
@@ -29,8 +29,7 @@
 #
 
 from __future__ import print_function
-from distutils import archive_util, dir_util
-from xml.etree.ElementTree import ElementTree
+
```
Comment (dictoon) [inline scripts/appleseed.package.py]: What's the rationale behind moving these imports around?

Comment (jdent02) [reply scripts/appleseed.package.py]: No rationale. I did a code format in PyCharm and it moved it

Comment (dictoon) [reply scripts/appleseed.package.py]: Doesn't make much sense to separate them like that :) Let's revert this. Feel free to merge the PR afterward (have you got the rights to do so?)

Comment (jdent02) [reply scripts/appleseed.package.py]: I’m pretty sure I do

**Pull request merged**